include/framework/everest.hpp

```diff
@@ -34,6 +34,49 @@ using TelemetryEntry = std::variant<std::string, const char*, bool, int32_t, uin
 using TelemetryMap = std::map<std::string, TelemetryEntry>;
 using UnsubscribeToken = std::function<void()>;
 
+enum class ErrorType {
```

Comment:
`ErrorType` might be a bit vague and could lead to confusions with the "error framework"

Reply:
Maybe call this `CallCmdEvent` or `CmdEvent`